assets/tests/CodeCheckerService.js

```diff
@@ -25,10 +25,18 @@ tie.factory('CodeCheckerService', [
       return code.indexOf(functionDefinition) !== -1;
     };
 
+    var _checkIfClassDefinitionExistsInCode = function(className, code) {
+      var classDefinition = 'class ' + className + '(object):';
```

Comment:
You might need to be a bit more thorough, as this is valid python:

`class        testClass   ():
  
  @classmethod
  def doAThing(cls, num):
    return num + 5
    
print testClass.doAThing(2)`